src/rdb.c

```diff
@@ -3081,7 +3081,7 @@ int rdbLoadRioWithLoadingCtx(rio *rdb, int rdbflags, rdbSaveInfo *rsi, rdbLoadin
             } else if (!strcasecmp(auxkey->ptr, "redis-ver")) {
                 serverLog(LL_NOTICE, "Loading RDB produced by Redis version %s", (char *)auxval->ptr);
```

Comment:
```suggestion
                serverLog(LL_NOTICE, "Loading RDB produced by Valkey version %s", (char *)auxval->ptr);
```

Reply:
Actually, this log entry is for the field "redis-ver", meaning we're loading an RDB file produced by Redis. We have another field "valkey-ver" too that's checked in the else-if below.